ci/do_ci.ps1

```diff
@@ -10,6 +10,8 @@ $nproc = (Get-ComputerInfo).CsNumberOfLogicalProcessors
 
 $SRC_DIR = (Get-Item -Path ".\").FullName
 
+# Workaround https://github.com/bazelbuild/bazel/issues/18683
+$BAZEL_STARTUP_OPTIONS = "--output_base=C:\Out"
```

Comment:
Is this a hard-coded path for CI environment?

Reply:
yes, the paths are apparently getting too long for windows so this is a reasonable workaround for CI